proxy/hdrs/MIME.cc

```diff
@@ -473,7 +473,8 @@ mime_hdr_set_accelerator_slotnum(MIMEHdrImpl *mh, int32_t slot_id, uint32_t slot
 inline void
 mime_hdr_set_accelerators_and_presence_bits(MIMEHdrImpl *mh, MIMEField *field)
 {
-  int slot_id, slot_num;
+  int slot_id;
+  ptrdiff_t slot_num;
```

Comment:
We can remove ``slot_num < 0`` but ``slot_num >= MIME_FIELD_SLOTNUM_UNKNOWN`` is still needed.
Because the value of ``MIME_FIELD_SLOTNUM_UNKNOWN`` is 14, so if the slot_num is 15, we need to set ``MIME_FIELD_SLOTNUM_UNKNOWN`` insetead of the 15.